### PR TITLE
joongang.co.kr - Popup

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [X] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://www.joongang.co.kr/

### Add your comment and screenshots

- https://www.similarweb.com/website/joongang.co.kr/#traffic

<details>
<summary>Screenshot</summary>

Screenshot from 2022-11-14 11-52-55

</details>

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: AnnoyancesFilter/Popups/sections/push-notifications_specific.txt [hunk 1]
```diff
@@ -20,6 +20,7 @@ macwelt.de,pcwelt.de#%#//scriptlet("set-constant", "firebase.messaging", "noopFu
 !
 ! SECTION: Push notifications - Regular rules
 !
+joongang.co.kr###joongang_plus_card_area
```
Comment (zloyden) [inline AnnoyancesFilter/Popups/sections/push-notifications_specific.txt:23]: I'm not sure that this is a push-notification. Looks like self-promo, but it's placed in header.
I don't think we should block it, but other annoyances which you can find in the content, you can.
@Alex-302 please also check it.

Comment (piquark6046) [reply AnnoyancesFilter/Popups/sections/push-notifications_specific.txt:23]: @zloyden It seems that it should be moved to `popups_specific.txt`.

Comment (zloyden) [reply AnnoyancesFilter/Popups/sections/push-notifications_specific.txt:23]: Anyway, they mention this in the menu, so I'd add this and other annoyances to `self-promo.txt`.

Comment (zloyden) [reply AnnoyancesFilter/Popups/sections/push-notifications_specific.txt:23]: `Other annoyances`
<details><summary>Screenshots:</summary>

image
image

</details><br/>

Comment (piquark6046) [reply AnnoyancesFilter/Popups/sections/push-notifications_specific.txt:23]: In fact, the `.layer_dark_popup.plotting_popup.login_proposal.plotting_mini.for_login` means a suggestion that requests a visitor that the visitor login to JoongAng website for unblocking some features without a subscription.
And, others request a visitor that the visitor subscribe a subscription of JoongAng.

Comment (zloyden) [reply AnnoyancesFilter/Popups/sections/push-notifications_specific.txt]: `회원에게만 제공되는 편의 기능` google translates this as `Convenience features available only to members`.
Other annoyances promote their `The JoongAng Plus` service.

Comment (zloyden) [reply AnnoyancesFilter/Popups/sections/push-notifications_specific.txt]: Ok, thanks for the explanation.
> the floating element related to member 

You can add it to the `popups_specific.txt` - it counts as floating element.
Others to `self-promo.txt`.

Comment (piquark6046) [reply AnnoyancesFilter/Popups/sections/push-notifications_specific.txt]: The `.goto-app-btn` element can be seen when you use mobile version of https://www.joongang.co.kr.
The `.banner_plus` exists on https://www.joongang.co.kr/article/25117599.

Comment (piquark6046) [reply AnnoyancesFilter/Popups/sections/push-notifications_specific.txt]: @zloyden

**Review by Alex-302 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**